README.md

```diff
@@ -138,6 +138,9 @@ require("typescript-tools").setup {
     -- described below
     tsserver_format_options = {},
     tsserver_file_preferences = {},
+    -- mirror of https://github.com/microsoft/vscode/blob/885dba39a5f546c5077196a8c31b03d03a293b31/extensions/typescript-language-features/package.json#L145-L1266
+    -- `typescript-tools.nvim` currently only supports `typescript.suggest.completeFunctionCalls`
+    vscode_configuration = {},
```

Comment:
I don't think it need to be called exactly like in vscode, I would prefer simple:
```lua
-- mirror of  vscode `typescript.suggest.completeFunctionCalls` option
-- maybe some url to vscode option docs
complete_function_calls = false,
```

We probably never will support all of vscode options simply because a lot of them are just proxies to `tsserver_format_options` and `tsserver_file_preferences` which are exposed now, so I prefer cherrypicking stuff.

Reply:
Sounds good, applied.